src/pages/members/[slug].astro

```diff
@@ -0,0 +1,65 @@
+---
+import { Image } from 'astro:assets';
+import { staffMembers } from '~/data/mockStaffMembers';
+import Layout from '~/layouts/MembersLayout.astro';
+import i18n from '~/utils/i18n';
+import bigImage from '~/assets/images/bigImage.png';
+import { Icon } from 'astro-icon/components';
+
+const { slug } = Astro.params;
+const member = staffMembers.find((m) => m.slug === slug);
+
+if (!member) {
+  throw new Error('Member not found');
+}
+
+export function getStaticPaths() {
+  return staffMembers.map((member) => ({
+    params: { slug: member.slug },
+  }));
+}
+
+const metadata = {
+  title: `Persian Flutter | ${member.firstName}`,
+  ignoreTitleTemplate: true,
+};
+
+i18n.setLang(Astro.currentLocale);
+---
+
+<Layout metadata={metadata}>
+  <div class="h-fit w-full relative flex flex-col items-center text-center mt-16 mb-[3rem] sm:mb-[10rem]">
+    <Image
+      src={bigImage}
+      alt="picture"
+      class="w-[90vw] sm:w-[70vw] md:w-[60vw] lg:w-[50vw] object-contain rounded-t-3xl sm:rounded-3xl sm:border-4 border-white sm:shadow-2xl shadow-black/25"
+    />
+    <div
+      class="sm:absolute sm:bottom-0 h-fit w-[90vw] sm:w-[50vw] md:w-[40vw] xl:max-w-fit bg-white/80 flex flex-col items-center gap-2 rounded-b-3xl sm:rounded-3xl text-lg p-8 shadow-xl shadow-black/5 backdrop-blur-xl border border-slate-300 sm:border-white sm:translate-y-1/2 z-[2]"
+    >
+      <h3 class="text-pfColorFirst font-bold text-xl">{`${member.firstName} ${member.lastName}`}</h3>
+      <p class="text-pfColorSecond font-medium">{member.jobTitle}</p>
+      <p class="text-black font-normal text-wrap mt-2">{member.description}</p>
+      <div class="flex justify-center items-center gap-3 mt-4 *:size-6 *:text-slate-500 dark:*:text-slate-500">
+        <Icon name="tabler:brand-telegram" />
+        <Icon name="tabler:brand-github" />
+        <Icon name="tabler:brand-linkedin" />
+      </div>
+    </div>
+  </div>
+  <div class="w-[80vw] flex flex-col gap-10 text-slate-700 dark:text-slate-400 text-lg mb-12">
+    <p>{member.aboutMe}</p>
+    <div>
+      <h4 class="font-bold text-pfColorFirst pb-3">تخصص‌های من</h4>
+      <ul class="list-disc list-inside">{member.skills.map((skill) => <li>{skill}</li>)}</ul>
+    </div>
+    <div>
+      <h4 class="font-bold text-pfColorFirst pb-3">علاقه‌مندی‌ها</h4>
+      <p>{member.favorites}</p>
+    </div>
+    <div>
+      <h4 class="font-bold text-pfColorFirst pb-3">ارتباط با من</h4>
+      <p>{member.contact}</p>
+    </div>
+  </div>
```

Comment:
About this part just as a reminder. we will use Content collection which I'm wokring on it in another branch. so basicly this section will be shown base on an .md file. do you have any Idea which we can prepare this section to work with markdown file?